inginious/frontend/task_dispensers/combinatory_test.py

```diff
@@ -49,10 +55,10 @@ def render_edit(self, template_helper, element, task_data, task_errors):
 
         taskset = element if isinstance(element, inginious.frontend.tasksets.Taskset) else None
         course = element if isinstance(element, inginious.frontend.courses.Course) else None
+        task_config = dict_data_datetimes_to_str(self._task_config)
```

Comment:
I'm wondering if this could not rather be implemented as a Jinja filter rather a recursive function here.

Reply:
The task_config dict is transformed into a json on the frontend by the jinja filter "tojson". Datetime objects being not serializable we have to transform those datetimes into strings before this tojson transformation. However it doesn't seem there are appropriate jinja filters to do this. Not in a efficient way at least.